src/test/java/net/snowflake/ingest/streaming/internal/RowBufferTest.java

```diff
@@ -602,6 +604,31 @@ public void testStringLength() {
     testStringLengthHelper(this.rowBufferOnErrorAbort);
   }
 
+  @Test
+  public void testRowIndexWithMultipleRowsWithErrorr() {
```

Comment:
Could you just update existing tests to verify the rowIndex? We have a few, search for `.getInsertErrors()`

Reply:
The existing tests verify other behaviors, I prefer having each test does its own thing. 

Also, we needed a good row and a bad row for that, hence I kept it in a different test.